src/common/components/repository-row/index.js

```diff
@@ -281,7 +281,7 @@ export class RepositoryRowView extends Component {
 
     const registeredName = snap.storeName;
 
-    const hasBuilt = !!(latestBuild && snap.snapcraftData);
+    const isBuilt = !!(latestBuild && snap.snapcraftData);
```

Comment:
I think `snapcraftData` is (or may be) `null` if snapcraft.yaml is not present.
So I guess `(latestBuild && typeof snap.snapcraftData !== null)`... unless we need to support both `null` and `undefined`...